app/views/layouts/_loggedin_header.html.erb

```diff
@@ -0,0 +1,35 @@
+<nav id = "login-navbar" class = "navbar navbar-default navbar-static-top navbar-fixed-top row">
+  <!-- responsive mobile stuff collapses down  -->
+  <div class="navbar-header col-xs-2 col-xs-offset-1">
+    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+    </button>
+    <a class="vertical-align col-xs-4" id="logo-link" href="#">
+      <img class="vertical-align" id="brand-pic" src='<%= asset_path('flag.png') %>'
+           alt="circular danish flag">
+    </a>
+    <!--company brand link -->
+    <%= link_to new_user_path, class: 'navbar-brand col-xs-12', id: 'brand-link' do %>
+      <p class="col-xs-4 vertical-align" id="brand-text">Danebook</p>
```

Comment:
When I'm signed in and click this, it takes me back to the home page, and then when I click on the user's name I get a 404.